### ISSUE TITLE
Add more instructions to the README

### ISSUE THREAD

**Issue by andykuszyk** [form3tech-oss/f1#4]
- [ ] Configuring Prometheus push gateway
 - [ ] Example directory

**Comment by markhowardform3** [comment]
Possibly something from https://github.com/form3tech/k6-tests/pull/182/files ?

Would be good to highlight the key features of `f1` compared to other tools:

- Testing multi-modal async processes - REST calls with messaging callbacks in our primary case. Scenarios written in go help us write test scenarios for these processes quickly, taking advantage of all go libraries nad test tools. 
- constant load injection - we inject load at a sustained rate, rather than slowing down as the system slows down like virtual user based systems will do
- Variable load profiles - e.g. gaussian

**Comment by akosveres** [comment]
I'd be very interested in the push gateway. Is it just about setting up an env variable? That's what I've seen in the test, but then there's an if clause and the `run()` is modified. So, if we could get some clarification, that would be great.

**Comment by andykuszyk-form3** [comment]
@akosveres yes, you should just need to set the `PROMETHEUS_PUSH_GATEWAY` environment variable with the address of your push gateway, and then metrics will be pushed :+1:
